core/src/main/scala/cromwell/core/simpleton/WomValueBuilder.scala

```diff
@@ -143,13 +151,13 @@ object WomValueBuilder {
       case arrayType: WomArrayType =>
         WomArray(arrayType, components.asArray map { toWomValue(arrayType.memberType, _) })
       case mapType: WomMapType =>
-        // map keys are guaranteed by the WDL spec to be primitives, so the "coerceRawValue(..).get" is safe.
+        // map keys are guaranteed by the WDL spec to be primitives, so the "coerceRawValue(..).get" is safe if this is WDL.
```

Comment:
What happens if this is not WDL?

Reply:
Yeah good question. This file was renamed from `WdlValueBuilder` and much of it wasn't updated for our brave new WOMmy world.